bus/bus.go

```diff
@@ -390,7 +390,10 @@ func (b *Bus) hostsetsContractsHandler(jc jape.Context) {
 
 func (b *Bus) objectsKeyHandlerGET(jc jape.Context) {
 	if strings.HasSuffix(jc.PathParam("key"), "/") {
-		jc.Encode(ObjectsResponse{Entries: b.os.List(jc.PathParam("key"))})
+		keys, err := b.os.List(jc.PathParam("key"))
+		if jc.Check("couldn't list objects", err) == nil {
```

Comment:
`uber nit` personally prefer to do `!= nil` and `return` to have consistent behaviour everywhere, we had several cases where due to copy paste the logic was inverted